contentcuration/contentcuration/tests/test_exportchannel.py

```diff
@@ -48,15 +56,19 @@ def license_wtfpl():
 
 
 def fileobj_video():
-    randomfilebytes = "4"
+    filecontents = "".join(random.sample(string.printable, 20))
```

Comment:
Not sure, but are there any tests that rely on predictable bytes to check hashes?